scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -236,22 +228,23 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "loss: 1.1876168941962533\n",
-      "hm_loss: 0.06973892802037085\n",
-      "wh_loss: 1.6554905762241106\n",
-      "off_loss: 0.19315332370922533\n",
-      "id_loss: 0.8924897752876859\n",
-      "time: 35.583333333333336\n"
+      "loss: 1.1128346400433464\n",
+      "hm_loss: 0.06353224289612051\n",
+      "wh_loss: 1.57920023114543\n",
+      "off_loss: 0.18636367223715702\n",
+      "id_loss: 0.8860541224528692\n",
+      "time: 44.016666666666666\n",
+      "Model saved to ./models/fairmot_ft.pth\n"
      ]
     }
    ],
    "source": [
-    "tracker.fit(num_epochs=EPOCHS, lr=LEARNING_RATE)"
+    "tracker.fit(num_epochs=EPOCHS, lr=LEARNING_RATE, resume=True)"
```

Comment:
This resume=True seems odd since we are starting to train on a new dataset.

Reply:
Because we default to finetuning the all_dla34 model, which is frozen at epoch 10